src/components/DangerButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, ButtonProps, createMuiTheme, MuiThemeProvider } from '@material-ui/core';
+import theme from '../themes/mainTheme';
```

Comment:
isteden for å importe og exporte themen, så kan du bruke hooken også:

```
import { useTheme } from '@material-ui/core/styles';

  const theme = useTheme();
```

Reply:
Hvis vi gjør det må vi opprette themet inni komponeten også da, og det vil da opprettes på nytt hver gang denne rendres, så tror det er best å legge den utenfor likevel